evadb/configuration/bootstrap_environment.py

```diff
@@ -46,7 +48,14 @@ def get_base_config(evadb_installation_dir: Path) -> Path:
 
 
 def get_default_db_uri(evadb_dir: Path):
-    return f"sqlite:///{evadb_dir.resolve()}/{DB_DEFAULT_NAME}"
+    import yaml
+
+    f = open(Path(EvaDB_INSTALLATION_DIR) / "evadb.yml", "r+")
+    config_obj = yaml.load(f, Loader=yaml.FullLoader)
+    if config_obj["experimental"]["use_postgres_backend"] is False:
+        return f"sqlite:///{evadb_dir.resolve()}/{DB_DEFAULT_NAME}"
+    else:
+        return f"postgresql://postgres:password@localhost:5432/{PG_DB_DEFAULT_NAME}"
```

Comment:
Can we move this string to `catalog_database_uri` in `evadb.yml`?

Reply:
Added fields in the yml file to specify the username, password, host and port instead. This gives the clients a more flexible option to connect.